packages/ui/components/empty-screen/EmptyScreen.tsx

```diff
@@ -65,11 +65,7 @@ export function EmptyScreen({
         )}
         {!customIcon ? null : <>{customIcon}</>}
         <div className={`flex ${limitWidth ? "max-w-[420px]" : ""}  flex-col items-center`}>
-          <h2
-            className={classNames(
-              "text-semibold font-cal text-emphasis text-center text-xl",
-              icon && "mt-6"
-            )}>
+          <h2 className={classNames("text-emphasis text-center text-xl font-bold", icon && "mt-6")}>
```

Comment:
why did you remove the font?

Reply:
font-cal was causing spacing between letters.I tried to reduce letter but was not working so i removed that font

Reply:
the issue is not font-cal the issue is that tremor has a "numeric" class above